ehrql/tables/tpp.py

```diff
@@ -552,6 +553,157 @@ class clinical_events_ranges(EventFrame):
     )
 
 
+@table
+class covid_therapeutics(EventFrame):
+    """
+    The COVID Therapeutics dataset contains information on COVID treatments used in inpatient
+    and outpatient settings.
+
+    **Metadata**
+
+    * **Data provider** NHS England
+    * **Participation / Coverage** Inpatients and outpatients treated with antivirals/nMABs for COVID-19 in England
+    * **Provenance** Data sourced largely from BlueTeq system (forms completed by clinicians)
+    * **Update frequency in OpenSAFELY** Approximately weekly
+    * **Delay between event occurring and event appearing in OpenSAFELY** Approximately 2-9 days
+    * **Collected information** Treatment start date; therapeutic intervention; COVID indication, current status, risk group, region
+
+
+    **Overview**
+
+    Antivirals and neutralising monoclonal antibodies (nMABs) for COVID-19 can be
+    administered in inpatient setting or, for outpatients, in COVID Medicine Delivery
+    Units (CMDUs) specifically set up for this purpose. For patients considered for
+    these treatments, clinicians submit completed forms to NHS England. Each row
+    represents one completed form for one course of treatment. Data received by
+    OpenSAFELY currently covers patients who were approved for treatment. The patient
+    may or may not have actually received the treatment or completed the course (but we
+    assume that they usually do). They may have another form completed for another
+    treatment, either because it was decided to give them a different treatment, or for
+    some other reason. They may in theory also have another form completed some months
+    later for another instance of infection.
+
+    Treatment dates may be in the past or future at the point when the form is
+    submitted.
+
+    Note that this dataset contains some **duplicate** rows – some full duplicates and
+    some partial duplicates.
+
+
+    **More Information**
+
+    * [Treatment guidelines](https://www.nice.org.uk/guidance/ta878)
+    * [Draft Data Report](https://docs.google.com/document/d/15o4x9sqHEO-sLm2dTqgm3PyAh72cdgOOmZC4AB3BTNk/) (currently only available to internal staff)
+    """
+
+    age_at_received_date = Series(
```

Comment:
We don't want to expose all of these fields - will need to check with Millie which fields should be exposed. From cohort-extractor, I think it would be:
- covid_indication
- intervention
- current_status
- received
- risk_cohort
- treatment_start_date
- region

Reply:
ok, if you could check with Millie & let me know for definite I will remove the unnecessary fields.

Reply:
Confirmed by Millie; just expose these fields :point_up:

Reply:
I have `risk_cohort` in place of `risk_group`, but otherwise have limited to this list of fields. I've kept all the fields in the raw table because of `SELECT DISTINCT` - they /probably/ could be removed, but I think the safest thing is to keep them?